doc/doc/basics/extractors.md

```diff
@@ -17,10 +17,15 @@ class contains different extractor *styles*:
 
 - Row-wise extractors:
   - [`json_extractor`](#json_extractor): highly flexible and compatible with
-  previous systems, but with limited performance 
+  previous systems, but with limited performance
```

Comment:
Feng, with strong opinions about tools? At least I have no strong opinions
:)

Jaeho is the boss :)

On Fri, Jun 12, 2015 at 2:18 PM, alldefector <notifications@github.com>
wrote:

> In doc/doc/basics/extractors.md
> <https://github.com/HazyResearch/deepdive/pull/319#discussion_r32355881>:
>
> > @@ -17,10 +17,15 @@ class contains different extractor *styles*:
> >
> >  - Row-wise extractors:
> >    - [`json_extractor`](#json_extractor): highly flexible and compatible with
> > -  previous systems, but with limited performance
> > +  previous systems, but with limited performance
>
> Start adopting Phabricator today, and it'll become the contributor's
> responsibility (as opposed to the maintainer's) to handle conflicts (rebase
> etc.)! :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HazyResearch/deepdive/pull/319/files#r32355881>.
>